ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/DataColumnSidecarGossipValidator.java

```diff
@@ -259,8 +274,13 @@ public SafeFuture<InternalValidationResult> validate(final DataColumnSidecar dat
     /*
      * [REJECT] The sidecar's column data is valid as verified by verify_data_column_sidecar_kzg_proofs(sidecar).
      */
-    if (!miscHelpersFulu.verifyDataColumnSidecarKzgProof(kzg, dataColumnSidecar)) {
-      return completedFuture(reject("DataColumnSidecar does not pass kzg validation"));
+    try (MetricsHistogram.Timer ignored =
+        dataColumnSidecarKzgBatchVerificationTimeSeconds.startTimer()) {
+      if (!miscHelpersFulu.verifyDataColumnSidecarKzgProof(kzg, dataColumnSidecar)) {
+        return completedFuture(reject("DataColumnSidecar does not pass kzg validation"));
+      }
+    } catch (final Throwable t) {
+      throw new RuntimeException(t);
```

Comment:
Do the same: `        return completedFuture(reject("DataColumnSidecar does not pass kzg validation"));
`